### PR TITLE
add TOF calib infos

### PR DESCRIPTION
**Pull request by noferini**

### CODE REVIEW

**Comment by noferini** [comment]
@sawenzel 
I added the steps needed to produce TOF calibration info without touching previous workflow

File: MC/bin/o2dpg_sim_workflow.py [hunk 1]
```diff
@@ -496,18 +496,23 @@ def createRestDigiTask(name, det='ALLSMALLER'):
...
+   TOFTPCMATCHERtaskCalib = createTask(name='tofitstpcmatch_'+str(tf), needs=[TOFRECOtask['name'], TPCRECOtask['name']], tf=tf, cwd=timeframeworkdir, lab=["RECO"], mem='1000')
+   TOFTPCMATCHERtaskCalib['cmd'] = 'o2-tof-matcher-workflow ' + getDPL_global_options() + ' --output-type calib-info --track-sources ITS-TPC'
+   workflow['stages'].append(TOFTPCMATCHERtaskCalib)
+
    MFTRECOtask = createTask(name='mftreco_'+str(tf), needs=[det_to_digitask["MFT"]['name']], tf=tf, cwd=timeframeworkdir, lab=["RECO"], mem='1500')
    MFTRECOtask['cmd'] = 'o2-mft-reco-workflow ' + getDPL_global_options(nosmallrate=False)
    workflow['stages'].append(MFTRECOtask)
 
-   pvfinderneeds = [ITSTPCMATCHtask['name'], FT0RECOtask['name'], TOFTPCMATCHERtask['name'], MFTRECOtask['name'], TRDTRACKINGtask['name']]
+   pvfinderneeds = [ITSTPCMATCHtask['name'], FT0RECOtask['name'], TOFTPCMATCHERtask['name'], TOFTPCMATCHERtaskCalib['name'], MFTRECOtask['name'], TRDTRACKINGtask['name']]
```
Comment (sawenzel) [inline MC/bin/o2dpg_sim_workflow.py:515]: why does pvfinding depend on TOFTPC calibration?

Comment (noferini) [reply MC/bin/o2dpg_sim_workflow.py]: You are right, my mistake.
Fixed

**Review by sawenzel — APPROVED** [review]

**Comment by sawenzel** [comment]
@noferini : I am afraid this PR broke the simulation chain. I am getting the following error:
```
44876:TOFClusterWriter]: [21:53:19][STATE] Exiting FairMQ state machine
[INFO] Dumping used configuration in dpl-config.json
[ERROR] invalid workflow in o2-tof-cluster-calib-workflow: No matching output found for TOF/INFOCALCLUS/0 as requested by data processor "TOFCalClusInfoWriter". Candidates:
```
in `tofmatch_1.log`

**Comment by sawenzel** [comment]
You could try with this `https://gitlab.cern.ch/aliperf/alibibenchtasks/-/blob/master/O2DPG_pp_minbias.sh` (taking only 1 timeframe and fewer events ... say 100 or 200).

**Comment by sawenzel** [comment]
@noferini : I have reverted this commit in the master. We'll put it back once it works in the given workflow.

**Comment by noferini** [comment]
Ok. I am rebuild o2 to the last dev since the script was not working. More news asap

**Pull request merged**